locales/tr/tr.json

```diff
@@ -1,618 +1,618 @@
-    "Guinea": "Guinea",
-    "Guinea-Bissau": "Guinea-Bissau",
+    "Guinea": "Gine",
+    "Guinea-Bissau": "Gine-Bissau",
     "Guyana": "Guyana",
     "Haiti": "Haiti",
-    "Heard Island & Mcdonald Islands": "Heard Island and McDonald Islands",
+    "Heard Island & Mcdonald Islands": "Heard Adası ve McDonald Adaları",
     "Hello!": "Merhaba!",
-    "Hide Content": "Hide Content",
-    "Hold Up!": "Hold Up!",
+    "Hide Content": "İçeriği Gizle",
+    "Hold Up!": "Bekle!",
     "Holy See (Vatican City State)": "Vatican City",
     "Honduras": "Honduras",
     "Hong Kong": "Hong Kong",
-    "Hungary": "Hungary",
-    "I agree to the :terms_of_service and :privacy_policy": "I agree to the :terms_of_service and :privacy_policy",
-    "Iceland": "Iceland",
-    "ID": "ID",
-    "If necessary, you may log out of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password.": "If necessary, you may log out of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password.",
+    "Hungary": "Macaristan",
+    "I agree to the :terms_of_service and :privacy_policy": ":terms_of_service ve :privacy_policy'yi kabul ediyorum",
+    "Iceland": "İzlanda",
+    "ID": "KİMLİK",
+    "If necessary, you may log out of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password.": "Gerekirse, tüm cihazlarınızda diğer tüm tarayıcı oturumlarınızdan çıkış yapabilirsiniz. Son oturumlarınızdan bazıları aşağıda listelenmiştir; ancak, bu liste kapsamlı olmayabilir. Hesabınızın ele geçirildiğini düşünüyorsanız, şifrenizi de güncellemelisiniz.",
     "If necessary, you may logout of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password.": "Gerekirse, tüm cihazlarınızdaki diğer tüm tarayıcı oturumlarınızdan çıkış yapabilirsiniz. Son oturumlarınızın bazıları aşağıda listelenmiştir; ancak bu liste kapsamlı olmayabilir. Hesabınızın ele geçirildiğini düşünüyorsanız, şifrenizi de güncellemelisiniz.",
-    "If you already have an account, you may accept this invitation by clicking the button below:": "If you already have an account, you may accept this invitation by clicking the button below:",
+    "If you already have an account, you may accept this invitation by clicking the button below:": "Zaten bir hesabınız varsa, aşağıdaki düğmeyi tıklayarak bu daveti kabul edebilirsiniz:",
```

Comment:
`"If you already have an account, you may accept this invitation by clicking the button below:": "Zaten bir hesabınız varsa, aşağıdaki butonu tıklayarak bu daveti kabul edebilirsiniz:",`